### ISSUE TITLE
Keybind pls :)

### ISSUE THREAD

**Issue by LeGerbi** [bl4sterino/Elytra-Drag#4]
Hey there

Really enjoy what you did here, can you add a keybind option so I can use it together with other elytra keybinds (like the boost in Deeper and Darker's elytra )? 

Cheers

**Comment by rompeballer** [comment]
I was about to say the same. I agree, would definitely be a nice feature!

**Comment by mechanira** [comment]
Yes! It would be great if you could rebind it to a more sensible key, like 'S'
